tests/Jellyfin.Server.Integration.Tests/Controllers/UserLibraryControllerTests.cs

```diff
@@ -23,7 +23,7 @@ public UserLibraryControllerTests(JellyfinApplicationFactory factory)
     }
 
     [Fact]
-    public async Task GetRootFolder_NonExistenUserId_NotFound()
+    public async Task GetRootFolder_NonexistentUserId_NotFound()
```

Comment:
`existent` was misspelled here (and in other instances below).

Note that nonexistent is a word in its own right which is appropriate in these cases which is why I'm selecting it -- but if you'd rather a smaller change that only added a `t` as required, I can provide that instead.